test/behavior/this_stage1.zig

```diff
@@ -0,0 +1,34 @@
+const expect = @import("std").testing.expect;
```

Comment:
All the tests in this file are passing in stage2; you can delete this file. The fact that you simplified the test case to not exercise struct initialization is fine; that's not what that test was testing. There's plenty of test coverage of struct initialization syntax in the struct.zig test cases.

Reply:
Makes sense, thanks!